### ISSUE TITLE
[REQ] Guide / Tutorial On How To Make Your Own FastFlags For Bloxstrap In The Menu. Like A Doc 

### ISSUE THREAD

**Issue by luastatee** [bloxstraplabs/bloxstrap#4157]
### Acknowledgement of preliminary instructions

- [X] I have read and acknowledged the preliminary instructions.

### What idea do you have?

i feel like people to want make their own bloxstrap fastflags (i know how) but you should make a doc on how to make your own fastflags

**Comment by Storm-930** [comment]
We can't make FastFlags. Only Roblox engineers can create them and we have no control over them, such as when one is released, changed, or deleted.

**Comment by luastatee** [comment]
yeah i just found that out sadly i know how to make fastflags but cant test them fully  to its max

**Comment by MrT-tak** [comment]
> yeah i just found that out sadly i know how to make fastflags but cant test them fully to its max

Unfortunately yeah.

@bluepilledgreat Please close.
(Unresolved, unplanned)

**Comment by MrT-tak** [comment]
> yeah i just found that out sadly i know how to make fastflags  but cant test them fully  to its max
> 

@luastatee How???
